src/plugins/plugin.legend.js

```diff
@@ -407,6 +407,7 @@ module.exports = function(Chart) {
 					}
 				};
 
+				var itemHeight = fontSize + labelOpts.padding;
```

Comment:
It'd be preferable to initialize this variable inside the `else` since it's not used until then. it makes it clearer that's the only place it's used

Reply:
Done.